src/order/modular_lattice.lean

```diff
@@ -338,3 +338,94 @@ instance complemented_lattice_Ici : complemented_lattice (set.Ici a) :=
 end complemented_lattice
 
 end is_modular_lattice
+
+
+section preorder
+
+variables [preorder α] {a b : α}
+
+open set
+
+lemma wcovby_iff_Ioo : a ⩿ b ↔ a ≤ b ∧ Ioo a b = ∅ :=
+and_congr_right' $ by simp [eq_empty_iff_forall_not_mem]
+
+lemma covby_iff_Ioo : a ⋖ b ↔ a < b ∧ Ioo a b = ∅ :=
+and_congr_right' $ by simp [eq_empty_iff_forall_not_mem]
+
+end preorder
```

Comment:
These will need to move to a much earlier file; probably there's a file about `Ioo` etc that this would go nicely in.

In the interest of not getting caught up in the mathlib4 port, I suggest you split these lemmas into their own PR so that it can be merged ASAP.

Reply:
`order.cover` will do.